.github/actions/send-slack-notification-slow-job/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Send a notification for exceeded build time
+description: Send a slack notification if build took more than planned
+
+inputs:
+  max-build-time-seconds:
+    description: Maximum acceptable build time in seconds
+    required: true
+  slack-token:
+    description: The slack token
+    required: true
+  slack-channel:
+    description: The slack channel id, channel name, or user id to post to
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Fetch billable time from gh API
+      id: fetch_time
+      shell: bash
+      run: |
+        total_time=$(gh api /repos/${{github.repository}}/actions/runs/${{github.run_id}}/timing | jq ' .billable.UBUNTU.total_ms /1000')
+        echo "total_time in seconds: ${total_time}"
+        echo "total_time=${total_time}" >> $GITHUB_OUTPUT
+
+    - name: Slack Notification
+      if: steps.fetch_time.outputs.total_time > inputs.max-build-time-seconds
```

Comment:
I wonder if this will be handled as a numerical comparison as both are strings: maybe use fromJson to convert (?) see https://stackoverflow.com/questions/68553401/how-to-make-mathematical-comparisons-in-github-action-expressions

Reply:
thanks, fixed